app/controllers/v0/debt_letters_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module V0
+  class DebtLettersController < ApplicationController
+    def index
+      render(json: service.list_letters)
+    end
+
+    def show
+      send_data(
+        service.get_letter(params[:id]),
```

Comment:
@camerontesterman do you think that we should verify that a debt letter is in the user's folder (by calling `list_letters` again), before allowing them to download the letter?